reframe/utility/os_ext.py

```diff
@@ -78,6 +80,7 @@ def run_command_async(cmd,
                             stderr=stderr,
                             universal_newlines=True,
                             shell=shell,
+                            env=env,
```

Comment:
Unfortunately, this is not equivalent to setting and unsetting the variable in `os.environ`. Quoting from `Popen`'s documentation:

> If env is not None, it must be a mapping that defines the environment variables for the new process; these are used instead of the default behavior of inheriting the current process’ environment.

Reply:
This explains the failure on Kesch and would lead to similar problems in other systems.

Reply:
Remove the `env` argument completely.